spec/models/user_spec.rb

```diff
@@ -1,23 +1,33 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
+  before (:each) do
+    @user = FactoryBot.create(:user)
+  end
   it "is creatable using a Factory" do
-    user = FactoryBot.create(:user)
-    expect(user).to be_valid
+    expect(@user).to be_valid
   end
 
   it "is not valid without an email" do
-    user = FactoryBot.build(:user, email: '')
-    expect(user).to_not be_valid
+    @user.email = ""
```

Comment:
Why do you use here "" and before in lecture testing nil ?

- [ ] check if it makes a difference if not would be good to be consistent.

Reply:
You are right. I adjusted the lectures test to use an empty string as well. I think an empty string is better suited for these tests because user input might always get converted/submited as an string and the database should handle this.